Contribute/get-started-setup-tools.md

```diff
@@ -21,52 +21,54 @@ This article describes the steps to interactively install Git client tools and V
-
-Markdown text is saved into files with .md extension.
-
-Additional details on how to write with Markdown, including Markdown basics and the features supported by Open Publishing Services (OPS) custom Markdown extensions, are covered in the [Markdown Reference](markdown-reference.md) article.
-
-## Visual Studio Code
+## Install Visual Studio Code
 
 [Visual Studio Code](https://code.visualstudio.com/), also known as VS Code, is a lightweight editor that works on Windows, Linux, and Mac. It includes git integration, and support for extensions.
 
-Download and install [VS Code](https://code.visualstudio.com/). The VS Code home page should detect your operating system correctly.
+> [!TIP]
+> To launch VS Code and open the current folder, run the command `code .` in the command line or bash shell. If the current folder is part of a local git repo, the GitHub integration appears in Visual Studio Code automatically.
+
+### Install Visual Studio Code for Window, mac and Linux
 
-- [Windows](https://code.visualstudio.com/docs/setup/windows)
+- [Windows](https://code.visualstudio.com/). The VS Code home page should detect your operating system correctly.
 - [Mac](https://code.visualstudio.com/docs/setup/mac)
 - [Linux](https://code.visualstudio.com/docs/setup/linux)
 
-> [!TIP]
-> To launch VS Code and open the current folder, run the command `code .` in the command line or bash shell. If the current folder is part of a local git repo, the GitHub integration appears in Visual Studio Code automatically.
+## Install Visual Studio Code extensions
 
-## Docs Authoring Pack
-Install the Docs Authoring Pack for Visual Studio Code. This set of extensions includes basic authoring assistance for help when writing Markdown, and a preview feature, so that you can see what the Markdown looks like in the style of the docs.microsoft.com site.
+[!INCLUDE[Install Visual Studio Code extentsion](~/guide/help-content/includes/proc-contribute-install-vscode-extensions.md)]
 
-   Visit this [marketplace page](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack) and select **Install**, or search for `docsmsft.docs-authoring-pack` in your extensions list in the VS Code window. 
+### Install Docs Authoring Pack
 
-   The Docs Authoring Pack is accessible by pressing Alt+M inside of VS Code. The toolbar is hidden by default but can be shown. Edit the VS Code settings (Control+comma) and adding user setting `"markdown.showToolbar": true` to show the toolbar.
+[!INCLUDE[Install Docs Authoring Pack](~/guide/help-content/includes/proc-contribute-install-docs-authoring-pack.md)]
 
-   For more information, see the [Docs Authoring Pack](how-to-write-docs-auth-pack.md) page.
+## Understand Markdown editors
```

Comment:
It seems strange to introduce Markdown editors after the  VS Code installation instructions. Maybe most of this could be deleted, and just the short paragraph about the MD reference article could remain here?

Reply:
@tdykstra I found it odd as well... I agree, I will make those changes.